pkg/collector/autodiscovery/configresolver.go

```diff
@@ -26,7 +27,7 @@ var (
 		"pid":            getPid,
 		"port":           getPort,
 		"container-name": getContainerName,
-		"tags":           getOptTags,
```

Comment:
I think we should remove this. We need to call getTags every time in resolve anyway.

Reply:
And so yeah please call it in resolve: https://github.com/DataDog/datadog-agent/blob/6da57d4a17db0c687bd635c46b4f194cfa450b50/pkg/collector/autodiscovery/configresolver.go#L152-L153